lighthouse-core/audits/performance-budget.js

```diff
@@ -20,6 +20,8 @@ const UIStrings = {
     =1 {1 request}
     other {# requests}
    }`,
+  /** Table column header for how displaying how much larger the size of network requests were than a predetermined budget. */
```

Comment:
```suggestion
  /** Table column header displaying: how much greater the quantity of network requests were than a predetermined budget, how much larger the size of network requests were than a predetermined budget. */
```

Reply:
Integrated this with the column format from `i18n.js`